internal/portforward/service/start.go

```diff
@@ -69,6 +69,14 @@ func (s *Service) Start(ctx context.Context) (runError <-chan error, err error)
 		return nil, fmt.Errorf("writing port file: %w", err)
 	}
 
+	if s.settings.Command != "" {
+		err = s.runUpCommand(ctx, ports)
+		if err != nil {
+			_ = s.cleanup()
+			return nil, fmt.Errorf("running port forward command: %w", err)
+		}
```

Comment:
I wonder, if the command fails, should port forwarding fail to start, and possibly then crash the container 🤔 
Maybe it would make more sense to just log it as an error, especially given port forwarding is *working*, it's just the user provided script that failed (which is somehow likely to happen imo)